WallsAndHoles/mapview.h

```diff
@@ -10,6 +10,8 @@
 #include <QKeyEvent>
 #include <QWidget>
 #include <QRegion>
+#include "tilemap.h"
+#include "tiletemplateset.h"
```

Comment:
Same comment here as for editor.h: these headers should be placed in the file they are used.

It helps keep things clean, but it is not a big deal!